README.md

```diff
@@ -1 +1,24 @@
-# morse-decoder
\ No newline at end of file
+# Morse Decoder Ruby
+
+> Morse code decipher using Ruby. For a different result just put a different input. 
+## Authors
+
+👤 **Mauro Vasconcelos**
+
+- [GitHub](https://github.com/vanillainanutshell)
+- [LinkedIn](https://www.linkedin.com/in/vanillainanutshell/)
+
+👤 **Daniel Ufeli**
+
+- [GitHub](https://github.com/danielufeli)
+- [LinkedIn](https://www.linkedin.com/in/danielcode/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/GuilhermeGiachellin/Morse-Code-Ruby/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
```

Comment:
- [OPTIONAL] It will be nice to give visitors some clues on how to run your application.